src/styles/_common/_typography.scss

```diff
@@ -1,130 +1,69 @@
-$base-font-size: 16px; // Default font-size
-
-@function em($pixels, $context: $base-font-size) {
-  @if (unitless($pixels)) {
-    $pixels: $pixels * 1px;
-  }
-  @if (unitless($context)) {
-    $context: $context * 1px;
-  }
-
-  @return $pixels / $context * 1rem;
-}
-
-@function str-replace($string, $search, $replace: "") {
-	$index: str-index($string, $search);
-
-	@if $index {
-		@return str-slice($string, 1, $index - 1) + $replace + str-replace(str-slice($string, $index + str-length($search)), $search, $replace);
-	}
-
-	@return $string;
-}
-
-@mixin font-face($name, $path, $weight: null, $style: null, $exts: woff2 woff eot ttf svg) {
-	$src: null;
-
-	$extmods: (
-		eot: "?",
-		svg: "#" + str-replace($name, " ", "_")
-	);
-
-	$formats: (
-		otf: "opentype",
-		ttf: "truetype"
-	);
-
-	@each $ext in $exts {
-		$extmod: if(map-has-key($extmods, $ext), $ext + map-get($extmods, $ext), $ext);
-		$format: if(map-has-key($formats, $ext), map-get($formats, $ext), $ext);
-		$src: append($src, url(quote($path + "." + $extmod)) format(quote($format)), comma);
-	}
-
-	@font-face {
-		font-family: quote($name);
-		font-style: $style;
-		font-weight: $weight;
-		src: $src;
-	}
-}
-
-@include font-face(IntervalNextBold, fonts/intervalnext-bold, null, null, woff2 woff eot);
-@include font-face(IntervalNextBook, fonts/intervalnext-book, null, null, woff2 woff eot);
-@include font-face(IntervalNextBookit, fonts/intervalnext-bookit, null, null, woff2 woff eot);
-@include font-face(IntervalNextMedium, fonts/intervalnext-medium, null, null, woff2 woff eot);
-@include font-face(IntervalNextMediumit, fonts/intervalnext-mediumit, null, null, woff2 woff eot);
-@include font-face(IntervalNextReg, fonts/intervalnext-regular, null, null, woff2 woff eot);
-@include font-face(IntervalNextRegularit, fonts/intervalnext-regularit, null, null, woff2 woff eot);
-
 $display-text: em(48px);
 $headline: em(32px);
-$header: em(24px);
+$title: em(24px);
 $subheader: em(20px);
 $body: em(16px);
 $caption: em(12px);
 $small: em(9px);
 
-@mixin font($name: "IntervalNextReg") {
-    font-family: $name, sans-serif;
-}
-
-@mixin heading() {
+@mixin display() {
     @include font("IntervalNextReg");
-    font-size: $headline;
+    font-size: $display-text;
     color: color('text','title');
 }
-@mixin subheading() {
+@mixin headline() {
     @include font("IntervalNextReg");
-    font-size: em(18px);
-    color: color('text','subheading');
-}
-@mixin header-label() {
-    @include font("IntervalNextReg");
-    font-size: em(24px);
+    font-size: $headline;
+    color: color('text','title');
+
+    &--light {
+        @include font("IntervalNextLi");
+    }
 }
 @mixin title() {
```

Comment:
I think they're displayed this way to "group" them and separate from the "fonts" mixins, but not sure